### PR TITLE
Feature/twitch

### PR DESCRIPTION
**Pull request by TekaLishoo**
Twich parsing

### CODE REVIEW

File: dao/container_redis.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from di.controller_general import ControllerGeneral
+from config.config import Settings
+from fastapi_redis_cache import FastApiRedisCache
+from fastapi import Request, Response
+from fastapi import APIRouter
+from fastapi_pagination import Page
+
+
+class ContainerRedisCash:
+    """
```
Comment (achopik) [inline dao/container_redis.py]: RedisCache

File: parsing_scripts/lamoda_parser.py [hunk 2]
```diff
@@ -26,58 +26,76 @@ async def parse(self):
...
+        a_list = soup.find_all("a", attrs={"class": f"{self.category_class}"})
 
         # in case of one product can be in several categories we'll store visited product's links
         # to be sure not to visit one product twice
         visited_products = set()
-        for a in a_list[:10]:  # slice is for test version
-            actual_url = self.BASE_URL + a.attrs['href']
+        for a in a_list[:5]:  # slice is for test version
+            actual_url = self.BASE_URL + a.attrs["href"]
             actual_soup = await self.get_soup(actual_url)
```
Comment (achopik) [inline parsing_scripts/lamoda_parser.py:37]: try to minimize iterations while parsing, you can use XPATH for advanced element picking, read some articles

**Review by achopik — CHANGES_REQUESTED** [review]
Lgtm, except few minor things

File: dao/container_redis.py (hunk 1 above)
Comment (TekaLishoo) [reply dao/container_redis.py]: Done

**Pull request merged**